ArduCopter/mode_auto.cpp

```diff
@@ -409,8 +409,18 @@ bool ModeAuto::wp_start(const Location& dest_loc)
 void ModeAuto::land_start()
 {
     // set horizontal speed and acceleration limits
-    pos_control->set_max_speed_accel_xy(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
-    pos_control->set_correction_speed_accel_xy(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
+    
+    const float adjusted_acceleration  = (float)copter.g.land_accel_limit;
+
+    if (adjusted_acceleration > 0.0f){
+        pos_control->set_max_speed_accel_xy(wp_nav->get_default_speed_xy(),adjusted_acceleration );
+        pos_control->set_correction_speed_accel_xy(wp_nav->get_default_speed_xy(), adjusted_acceleration);
+    }
```

Comment:
Normally we put the else on the same line as the preceding curly brace.

Reply:
copied, I am going to ammend the same. Thanks